### PR TITLE
Add milkomeda algorand mainnet

### PR DESCRIPTION
**Pull request by pkakelas**
> Network Name: Milkomeda Algorand Mainnet
> New RPC URL: https://rpc-mainnet-algorand-rollup.a1.milkomeda.com
> Chain ID: 2002
> Currency Symbol: mALGO
> Block Explorer URL: https://explorer-mainnet-algorand-rollup.a1.milkomeda.com/

Solves: https://github.com/safe-global/safe-deployments/issues/133

Thanks folks :)

### CODE REVIEW

**Comment by moisses89** [comment]
Hi @pkakelas could you fill the following form to create the issue and link it with the PR? 
https://github.com/safe-global/safe-deployments/issues/new?assignees=&labels=%5Benhancement%5D&template=new_chain.yaml&title=%5BNew+chain%5D%3A+

**Comment by pkakelas** [comment]
Done!

File: src/assets/v1.3.0/proxy_factory.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "1287": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
     "1984": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
     "2001": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+    "2002": "0xE89ce3bcD35bA068A9F9d906896D3d03Ad5C30EC",
```
Comment (moisses89) [inline src/assets/v1.3.0/proxy_factory.json:45]: Could you review the addressess? some of them are not the deterministic addresses, this is one case. 
```
reusing "SimulateTxAccessor" at 0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da
deploying "GnosisSafeProxyFactory"
```

**Review by moisses89 — CHANGES_REQUESTED** [review]

**Comment by pkakelas** [comment]
Arghh sorry my bad! @moisses89 fixed it :)

**Review by Uxio0 — APPROVED** [review]
I checked and looks good now, but we cannot merge after @moisses89 approves

**Review by moisses89 — APPROVED** [review]
Looks good to me!

**Pull request merged**